### PR TITLE
CPR-519-add-indexes

### PR DESCRIPTION
**Pull request by JunaedAhmedMOJ**

### CODE REVIEW

File: src/main/resources/db/migration/V70__add_indexes_for_child_tables.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+BEGIN;
+-------------------------------------------------------
+CREATE INDEX idx_identifier_type ON reference(identifier_type);
```
Comment (markreesmoj) [inline src/main/resources/db/migration/V70__add_indexes_for_child_tables.sql:3]: The convention for our index names (which I think have to be unique in the database) is to use the table name

So these should `be idx_reference_identifer_type`

**Review by markreesmoj — COMMENTED** [review]
Please rename the indexes to include the table name

**Review by markreesmoj — APPROVED** [review]
👍

**Review by H-Iwanejko — COMMENTED** [review]
Just a question, should these be a combine index of type and value?

**Comment by JunaedAhmedMOJ** [comment]
> Just a question, should these be a combine index of type and value?

hmm i guess it would depend on if we always try and get both values from our query?

**Pull request merged**